Forta-Bot-Challenge-4/COMP01/src/constants.ts

```diff
@@ -0,0 +1,12 @@
+import { ethers } from "ethers";
+
+export const SUPPLY_EVENT_SIGNATURE: string = "event Supply(address indexed from, address indexed dst, uint amount)";
+
+export const WRONG_SUPPLY_EVENT_SIGNATURE: string =
+  "event fooSupply(address indexed from, address indexed dst, uint amount)";
+
+export const PROXY_CONTRACT_ADDRESS: string = "0xc3d688B66703497DAA19211EEdff47f25384cdc3";
+
+export const SUPPLY_POOL_ADDRESS: string = "0xc3d688B66703497DAA19211EEdff47f25384cdc3";
```

Comment:
```suggestion
export const PROXY_CONTRACT_ADDRESS: string = "0xc3d688B66703497DAA19211EEdff47f25384cdc3";

export const SUPPLY_POOL_ADDRESS: string = "0xc3d688B66703497DAA19211EEdff47f25384cdc3";
```
Addresses are the same, so one should be enough. Additionally, if you are going to have `PROXY` in the name, it would be helpful to add more to it so the name is more descriptive.